public/docs/_examples/component-styles/dart/lib/hero.dart

```diff
@@ -0,0 +1,6 @@
+class Hero {
+  String name;
+  List<String> team;
+  bool active;
```

Comment:
Empty line before the constructor:
https://github.com/angular/angular.io/wiki/Conventions-for-Dart-Samples